aws-frauddetector-detector/src/aws_frauddetector_detector/helpers/update_worker_helpers.py

```diff
@@ -20,7 +20,7 @@ def validate_dependencies_for_detector_update(
     afd_client, model: models.ResourceModel, previous_model: models.ResourceModel
 ):
     # TODO: revisit  this validation when/if we support in-place teardown
-    #   For now, throw bad request for unsupported event type update and validate external models
+    #   For now, throw bad request for unsupported event type update and external models, and model versions
```

Comment:
nit: Without the `validate` word, this seems worded oddly, since we're doing slightly different things with event type checks vs. model checks.

```
For now, throw bad request for unsupported event type update, and validate external models + model versions
```

Reply:
Good catch. Revising